tests/integration/test_advanced_indexing.py

```diff
@@ -177,9 +177,17 @@ def test_empty_bool():
     res_num = arr_num[idx_num]
     assert np.array_equal(res_np, res_num)
 
-    arr_np[idx_np] = val_np
-    arr_num[idx_num] = val_num
-    assert np.array_equal(arr_np, arr_num)
+    # TODO this doesn't work until this PR is merged:
+    # https://github.com/nv-legate/cunumeric/pull/473
+    # arr_np[idx_np] = val_np
+    # arr_num[idx_num] = val_num
+    # assert np.array_equal(arr_np, arr_num)
```

Comment:
this test doesn't work anymore because PointN type can't work when creating np.array(0). PointN type will be translated to one of the np.dtypes in the https://github.com/nv-legate/cunumeric/pull/473 so this test will work again

Reply:
I don't think the test will work still, because `fill` (namely the one you do on line 587 above) works only for fully-supported dtypes (e.g. np.float32 and np.int32).

Reply:
That is right. Actually we don't need to call `fill` for `set_item`, because `out` will not get used if it's size is 0 in this case